Hey can you my my app.js into the main line?

choefer-matc opened the pull request:
May not need the .env file, please review and let me know...

Commit:
Created app.js base on the book code....

reformatted to read it

and made a change and realize we have to restart the app to see the change.....

Commit:
Played with. env file.......

Had to run with this command line....

node --env=.env app.js

Commit:
Added date/time output

choefer-matc commented:
Hey can we get a simulated D6 roll feature in the app, before I accept your PR..

Commit:
Boilerplate D6 results

Still need to make them real, but wanted to test in render

Commit:
Real Random D6 feature render live test

choefer-matc committed:
Used this to get our pointy-clicky to get AWS deploy working..

Yes! ! !

choefer-matc committed:
Got it working after tunnle timeouts! phew! :-)

choefer-matc committed:
AI gave me a solution with timestamps!

Neato.. but is it idempotent?

tsteelematc commented:
Sorry I didn't see your AI commits... Timestamps with RANDOM make the script run over and over, without failing, but not idempotent, as it always creates new assets. Hope that makes sense. Thanks.

choefer-matc committed:
Moved app stuff to src folder....

Next work on the BASH stuff in a folder....

choefer-matc committed:
Moved BASH stuff to bash folder...

choefer-matc committed:
our first YAML from the book...

choefer-matc committed:
Run ansible_playbook.....

ansible $ ansible-playbook -v create_ec2_instances_playbook.yml

choefer-matc committed:
created ansible inventory yml..

choefer-matc committed:
Created group vars folder and new YML file...

choefer-matc committed:
Starting our config yaml..

choefer-matc committed:
Fancy way to setup symlink for app.js

ln -s~/sample-app/src/app.js ~/sample-app/ansible/roles/sample-app/files/app.js

choefer-matc committed:
Got main yml ready..

goona run in next vid..

choefer-matc committed:
Created packer folder and symlink FTW!

ln -s ~/sample-app/src/app.js ~/sample-app/packer/app.js

choefer-matc committed:
created sample-app.pkr.hcl file example frome textbook...

us-east-2 changed to us-east-1

choefer-matc committed:
Got the install-node.sh file for packer!!

creates a bash shell to install node...

choefer-matc committed:
Created first .tf  Tofu file..

choefer-matc committed:
Tofu user-data.sh

we've seen running our app with nohub before..

choefer-matc committed:
variables.tf ftw!!

choefer-matc committed:
Created outputs.tf

choefer-matc committed:
Got out first Tofu working ...and

committed appropriate files

choefer-matc committed:
Tested updating an EC2 with Tofu...

added a tag...

remember, sever still didn't work with our app because user-data.sh still wasn't running at this point.

choefer-matc committed:
Starting the modules example...

Moved some previous work into a
modules folder...

and started a live/main.tf

choefer-matc committed:
Got two instances running via OpenTofu and modules

via open Tufu and modules.

choefer-matc committed:
re-installed node and packer, put sym link back into packer and created new AMI with packer..

also changed app.js  to roll a D20
